python/tchannel/tornado/data.py

```diff
@@ -172,6 +177,13 @@ def status_code(self, status):
 
         self.flags = status.value
 
+    @property
+    def ok(self):
+        if self.flags == StatusCode.ok.value:
```

Comment:
Right, so it seems like this isn't right -- `flags` controls fragmentation, what we want is `code`: https://github.com/uber/tchannel/blob/master/docs/protocol.md#code1